docs/schema_markdown/AnalysisProcess.md

```diff
@@ -29,16 +29,16 @@ URI: [GHGA:AnalysisProcess](https://w3id.org/GHGA/AnalysisProcess)
      * Description: The name for an entity.
      * Range: [String](types/String.md)
  * [AnalysisProcess➞analysis](AnalysisProcess_analysis.md)  <sub>1..1</sub>
-     * Description: The Analysis the AnalysisProcess was part of
+     * Description: The Analysis the Analysis Process is part of.
      * Range: [Analysis](Analysis.md)
  * [AnalysisProcess➞study_input_files](AnalysisProcess_study_input_files.md)  <sub>0..\*</sub>
-     * Description: The StudyFile associated used as an input for an entity.
+     * Description: The associated Study File used as an input for this Analysis Process.
      * Range: [StudyFile](StudyFile.md)
  * [AnalysisProcess➞sample_input_files](AnalysisProcess_sample_input_files.md)  <sub>0..\*</sub>
-     * Description: The SampleFile associated used as an input for an entity.
+     * Description: The associated Sample File used as an input for this Analysis Process.
```

Comment:
Can we instead say for the corresponding analysis process - not sure if "this" precise?